server/backends/blobstore/blobstore.go

```diff
@@ -558,7 +564,7 @@ func (a *AwsS3BlobStore) upload(ctx context.Context, blobName string, compressed
 
 func (a *AwsS3BlobStore) DeleteBlob(ctx context.Context, blobName string) error {
 	start := time.Now()
-	err := a.delete(ctx, blobName)
+	err := a.delete(ctx, blobPath(blobName))
```

Comment:
nit: call `blobPath` in `delete()` instead

Reply:
fixed